lexicons/com/atproto/admin/defs.json

```diff
@@ -17,6 +17,10 @@
       "properties": {
         "id": { "type": "integer" },
         "action": { "type": "ref", "ref": "#actionType" },
+        "actionDurationInHours": {
```

Comment:
Using `Hours` because realistically, this would be smallest unit of time that we would use for reverting actions